mkosi/distributions/gentoo.py

```diff
@@ -301,7 +286,9 @@ def mkosi_conf(self) -> None:
         # we use this so we don't need to touch upstream files.
         # we also use this for documenting build environment.
         emerge_vars_str = ""
-        emerge_vars_str += "\n".join(f'{k}="${{{k}}} {v}"' for k, v in self.emerge_vars.items())
+        emerge_vars_str += "\n".join(
+            f'{k}="${{{k}}} {v}"' for k, v in self.emerge_vars.items()
+        )
```

Comment:
This is also just a formatting change.

Reply:
> This is also just a formatting change.

E501 line too long (size > 79 characters)

Reply:
Ditto, keep this a single line